addons/website_event_sale/static/tests/tours/helpers/WebsiteEventSaleTourMethods.js

```diff
@@ -7,15 +7,12 @@ odoo.define('website_event_sale.tour.WebsiteEventSaleTourMethods', function (req
                 content: "Go to page Shop",
                 trigger: '.nav-link:contains("Shop")',
             },
-            {
-                content: "Toggle Pricelist",
-                trigger: '.o_pricelist_dropdown > .dropdown-toggle',
-                run: 'click',
-            },
             {
                 content: `Activate Pricelist ${pricelistName}`,
-                trigger: `.dropdown-item:contains(${pricelistName})`,
-                run: 'click',
+                trigger: '.o_pricelist_dropdown',
+                run: function () {
+                    $(`.o_pricelist_dropdown .dropdown-item:contains("${pricelistName}")`)[0].click();
+                },
```

Comment:
Is the problem (only) on master?
Altogether it seems unrelated to the task or should be included in the related IMP if I missed it, but only if it is the right approach. (Why not solve the css issue?)
So maybe this commit should not be *here*?

Reply:
I understand the feeling, I'll be glad when this is over too.
I also think that if there is a problem with the tour, we can't just fix it here and not in earlier maintained releases. 

If it's only a warning and likely the same thing in all runbots, there is no need to do something now. If it is an error and the tour is being disabled on runbots because of it, I think you could create a separate task and a separate PR with this commit in the earliest version where it occurs.

There is going to be a review process for this task as well, that is going to take *unknown amount of* time, and when you'll rebase, this commit will naturally be removed from this PR.